src/common/convolution.cpp

```diff
@@ -183,8 +198,9 @@ status_t conv_attr_check(const convolution_desc_t &desc, const engine_t *engine,
             const int mask_dst = sc.get(DNNL_ARG_DST).mask_;
             const bool with_groups
                     = desc.src_desc.ndims != desc.weights_desc.ndims;
-            VCHECK_CONV_UNIMPL(utils::everyone_is(0, mask_src, mask_dst)
-                            && utils::one_of(mask_wei, 0, with_groups ? 3 : 1),
+            VCHECK_CONV_UNIMPL(utils::one_of(mask_wei, 0, with_groups ? 3 : 1)
+                            && utils::one_of(mask_dst, 0, 2)
+                            && utils::one_of(mask_src, 0, 3),
```

Comment:
Could you, please, clarify why mask=3 for src but mask=2 for dst?

Reply:
Leftover from work on per_oc src scales which was removed from scope, restored to original conditions for src, thanks.